### DISCUSSION TITLE
I would like to ask if the use of ClosedXML 0.76.0.0 is free of charge?

### DISCUSSION THREAD

**Discussion by PochengTaiwan** [ClosedXML/ClosedXML#2158] [Q&A]
As the title says:

I use ClosedXML 0.76.0.0 in .NET Framework 3.5 for web application development to allow users to upload Excel files and create and read them.

**Comment by igitur** [comment]
Yes, ClosedXML is licensed under the MIT license and is therefore free to use. Please read the full license for more details. https://github.com/ClosedXML/ClosedXML/blob/develop/LICENSE

**Reply by PochengTaiwan** [reply]
Sorry!! I'll ask again in more detail..

I quoted ClosedXML and provided it to the customer for use, without making any changes to the ClosedXML program, simply by referencing the DLL in c# and writing for reading and writing. Does this comply with the free use rules of MIT??

Thank you for your response. Your response will help the customer understand and use it with peace of mind.

Because the customer is really dumb, although the texts in MIT have already explained, they are stuck on how to prove it, so they have to ask questions to seek your advice.

**Reply by vbjay** [reply]
Read the license here  https://github.com/ClosedXML/ClosedXML/blob/develop/LICENSE
